clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/ModifyGoogleServerGroupInstanceTemplateAtomicOperation.groovy

```diff
@@ -83,13 +84,18 @@ class ModifyGoogleServerGroupInstanceTemplateAtomicOperation implements AtomicOp
     def region = description.region
     def serverGroupName = description.serverGroupName
     def serverGroup = GCEUtil.queryServerGroup(googleClusterProvider, accountName, region, serverGroupName)
+    def isRegional = serverGroup.regional
+    // Will return null if this is a zonal server group.
     def zone = serverGroup.zone
 
-    def instanceGroupManagers = compute.instanceGroupManagers()
+    def instanceGroupManagers = isRegional ? compute.regionInstanceGroupManagers() : compute.instanceGroupManagers()
     def instanceTemplates = compute.instanceTemplates()
 
     // Retrieve the managed instance group.
-    def managedInstanceGroup = instanceGroupManagers.get(project, zone, serverGroupName).execute()
+    def managedInstanceGroup =
```

Comment:
I started with that exact setup. I ended up switching to the current style because some of the operations require more setup than just the `def instanceGroupManager` and `def location` assignments. So, some of the operations ended up with 3 or 4 assignments, each with a ternary operator considering the same `isRegional` flag. I think while the current style is more verbose, it is very clear what is taking place, and ends up more readable.